forge/How to make a new extension 3 3/how-to-make-a-new-extension-3-3.md

```diff
@@ -5,8 +5,11 @@ updated_at: '2018-10-10 08:48:53'
 authors:
     - 'Christophe Garcia'
     - 'Martin Nicholson'
+contributors:
+    - 'Bartlomiej Marszal'
 tags:
-    - 'Legacy Versions:TAO 3.0'
+    - 'Extension Development'
+    - 'Onboarding'
 -->
```

Comment:
Please could you add an additional tag 'Tutorials' ?
Then the article should appear in both sections.

Reply:
Done